src/app/zones/views/ZoneCreateView.vue

```diff
@@ -298,49 +295,44 @@
-                        </template>
-                        <template
+                    <KEmptyState v-else>
+                      <template #icon>
+                        <ProgressIcon
+                          v-if="data === undefined"
+                          data-testid="waiting"
+                          display="inline-block"
+                          :color="KUI_COLOR_TEXT_NEUTRAL_WEAK"
+                        />
+
+                        <CheckCircleIcon
                           v-else
-                        >
-                          <CheckCircleIcon
-                            data-testid="connected"
-                            display="inline-block"
-                            :color="KUI_COLOR_TEXT_SUCCESS"
-                          />
-                          {{ t('zones.form.scan.completeTitle') }}
-                        </template>
+                          data-testid="connected"
+                          display="inline-block"
+                          :color="KUI_COLOR_TEXT_SUCCESS"
+                        />
                       </template>
-                      <template #message>
-                        <template v-if="(typeof data !== 'undefined')">
-                          <p>
-                            {{ t('zones.form.scan.completeDescription', { name }) }}
-                          </p>
-                          <p class="mt-2">
-                            <KButton
-                              appearance="primary"
-                              :to="{
-                                name: 'zone-cp-detail-view',
-                                params: {
-                                  zone: name,
-                                },
-                              }"
-                            >
-                              {{ t('zones.form.scan.completeButtonLabel', { name }) }}
-                            </KButton>
-                          </p>
-                        </template>
+
+                      <template #title>
+                        {{ data === undefined ? t('zones.form.scan.waitTitle') : t('zones.form.scan.completeTitle') }}
+                      </template>
+
+                      <template v-if="(typeof data !== 'undefined')">
+                        <p>
+                          {{ t('zones.form.scan.completeDescription', { name }) }}
+                        </p>
+                        <p class="mt-2">
+                          <KButton
+                            appearance="primary"
+                            :to="{
+                              name: 'zone-cp-detail-view',
+                              params: {
+                                zone: name,
+                              },
+                            }"
+                          >
+                            {{ t('zones.form.scan.completeButtonLabel', { name }) }}
+                          </KButton>
+                        </p>
```

Comment:
Apart from removing the `cta-is-hidden`attribute and the switch from `#message` to `#default`, does this make any other changes?

Reply:
Yes: This is now using the `icon` slot instead of adding the icon to the title.